frontend/src/metabase/styled-components/layout/FullWidthContainer.tsx

```diff
@@ -15,7 +15,7 @@ export const FullWidthContainer = styled.div`
     padding-right: 1.5rem;
   }
 
-  ${breakpointMinMedium} {
+  ${breakpointMinLarge} {
```

Comment:
Would this affect other screens that are using `FullWidthContainer`, as their breakpoint has been changed?

Reply:
yes, but the effect should be quite minimal since there are not a whole lot of components that use `FullWidthContainer`.

![image](https://github.com/user-attachments/assets/0ec5aad7-c0f2-4b2e-8153-7fa2bbe6583d)